mwb.py

```diff
@@ -122,6 +146,21 @@ def main():
         shutil.rmtree(dir_output, ignore_errors=True)
         os.mkdir(dir_output)
 
+        # generate dict of filenames and their wikipaths
+        for root,dirs,files in os.walk(dir_wiki):
+            dirs[:]=[d for d in dirs if not d.startswith('.')]
+            files=[f for f in files if not f.startswith('.')]
+            readable_path = root[len(dir_wiki):]
+            path = scrub_path(readable_path)
+            for file in files:
+                if file in ['netlify.toml']:
+                    continue
+                clean_name = scrub_path(file)
+                if file.endswith('.md'):
+                    wikifiles[Path(file[:-3]).name] = f"{path}/{clean_name}"
```

Comment:
should use `with_suffix()` instead of `[:-3]`?

and probably use pathlib's slash operator rather than the f-string.